contracts/contracts/ccip/fee_quoter/contract.tolk

```diff
@@ -40,11 +40,20 @@ fun onInternalMessage(in: InMessage) {
             assert(st.allowedPriceUpdaters.exists(in.senderAddress) || st.ownable.owner == in.senderAddress, FeeQuoter_Error.UnauthorizedPriceUpdater);
             updatePrices(mutate st, msg.updates);
             st.store();
+
+            val returnValueResponse = createMessage({
```

Comment:
@krebernisak took a diff approach [here](https://github.com/smartcontractkit/chainlink-ton/pull/336/files#diff-b99e9ace8f8a7e6c44a76f9abbfea433ac58cfa54c1eb5c3c96734f4f36050e3R622) responding with a message with body.

As a follow up I think we should have consistency on managing excesses. May be in certain calls we should add an extra field to specify an address (transmitter) to send excesses back directly to transmitter.